src/gt4py/cli.py

```diff
@@ -62,15 +62,16 @@ def convert(
 
     @staticmethod
     def get_backend_names() -> KeysView:
-        return gt_backend.REGISTRY.keys()
+        return gt4py.backend.REGISTRY.keys()
 
     @staticmethod
     def enabled_backend_cls_from_name(backend_name: str) -> Optional[Type[CLIBackendMixin]]:
         """Check if a given backend is enabled for CLI."""
-        backend_cls = gt_backend.from_name(backend_name)
+        backend_cls = gt4py.backend.from_name(backend_name)
+        assert backend_cls is not None
         if not issubclass(backend_cls, CLIBackendMixin):
             return None
-        return backend_cls
+        return cast(Optional[Type[CLIBackendMixin]], backend_cls)
```

Comment:
Is the `Optional` typing needed here?
```suggestion
        return cast(Type[CLIBackendMixin], backend_cls)
```

Reply:
Nice catch!